frontend/src/pages/modelRegistry/screens/InvalidModelRegistry.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import EmptyStateErrorMessage from '~/components/EmptyStateErrorMessage';
+import ProjectSelectorNavigator from '~/concepts/projects/ProjectSelectorNavigator';
+
+type InvalidModelRegistryProps = {
+  title?: string;
+  modelRegistry?: string;
+  getRedirectPath: (namespace: string) => string;
+};
+
+const InvalidModelRegistry: React.FC<InvalidModelRegistryProps> = ({
+  title,
+  modelRegistry,
+  getRedirectPath,
+}) => (
+  <EmptyStateErrorMessage
+    title={title || 'Model Registry not found'}
+    bodyText={`${
+      modelRegistry ? `Model Registry ${modelRegistry}` : 'The Model Registry'
+    } was not found.`}
+  >
+    {/* TODO: Replace this with a model registry selector */}
```

Comment:
@lucferbux Do we have a JIRA for this, or will it be done in this PR?

Reply:
Good catch, I was waiting until das pr and I didn't add it, updating the PR